### ISSUE TITLE
F1ScoreCallback has wrong metric_fn

### ISSUE THREAD

**Issue by Podidiving** [catalyst-team/catalyst#1010]
## 🐛 Bug Report
https://github.com/catalyst-team/catalyst/blob/3126390f9f679ebcfedbe01707b416678a2732ac/catalyst/callbacks/metrics/f1_score.py#L5
I'm not sure, If it is a bug or not, but it seems to me that `F1ScoreCallback` is using wrong `F1ScoreCallback`
It is using this one 
https://github.com/catalyst-team/catalyst/blob/3126390f9f679ebcfedbe01707b416678a2732ac/catalyst/metrics/f1_score.py#L51
and not this one
```
def f1_score(
        outputs: torch.Tensor,
        targets: torch.Tensor,
        beta: float = 1.0,
        eps: float = 1e-7,
        threshold: float = None,
        activation: str = "Sigmoid",
):
    """
    Args:
        outputs (torch.Tensor): A list of predicted elements
        targets (torch.Tensor):  A list of elements that are to be predicted
        eps (float): epsilon to avoid zero division
        beta (float): beta param for f_score
        threshold (float): threshold for outputs binarization
        activation (str): An torch.nn activation applied to the outputs.
            Must be one of ["none", "Sigmoid", "Softmax2d"]

    Returns:
        float: F_1 score
    """
    activation_fn = get_activation_fn(activation)

    outputs = activation_fn(outputs)

    if threshold is not None:
        outputs = (outputs > threshold).float()

    true_positive = torch.sum(targets * outputs)
    false_positive = torch.sum(outputs) - true_positive
    false_negative = torch.sum(targets) - true_positive

    precision_plus_recall = (
            (1 + beta ** 2) * true_positive
            + beta ** 2 * false_negative
            + false_positive
            + eps
    )

    score = ((1 + beta ** 2) * true_positive + eps) / precision_plus_recall

    return score
```
As it should be, in my opinion

**Comment by Scitator** [comment]
🤔 @Podidiving nice catch! 
Would you like to contribute a fix for F1ScoreCallback callback compatibility with the new metric function?

**Comment by akatasonov** [comment]
@Scitator any workaround for the time being? It seems it is not possible to use F1ScoreCallback in the recent version of the Catalyst without this fix. Thank you for your great package!

**Comment by akatasonov** [comment]
ok, replying to my own question: I had to copy the code of the callback and make it is using the correct f1_score (and don't import it from Catalyst). Ugly, but it works

**Comment by Scitator** [comment]
Creating custom callbacks is... normal :)
We have various metrics supported - https://catalyst-team.github.io/catalyst/api/metrics.html, but not all of them support callbacks, so you need to write them on your own for your data :)
